ethereum/core/src/testFixtures/java/tech/pegasys/teku/core/BlockProposalTestUtil.java

```diff
@@ -98,6 +107,35 @@ public SignedBlockAndState createNewBlock(
     return new SignedBlockAndState(signedBlock, newBlockAndState.getState());
   }
 
+  private ExecutionPayload createExecutionPayload(
+      final UInt64 newSlot,
+      final BeaconState state,
+      final UInt64 newEpoch,
+      final Optional<List<Bytes>> transactions) {
+    final SpecVersion specVersion = spec.atSlot(newSlot);
+    final ExecutionPayloadSchema schema =
+        SchemaDefinitionsMerge.required(specVersion.getSchemaDefinitions())
+            .getExecutionPayloadSchema();
+    if (transactions.isEmpty()) {
+      return schema.getDefault();
+    }
+    return schema.create(
+        BeaconStateMerge.required(state).getLatestExecutionPayloadHeader().getBlockHash(),
+        Bytes20.ZERO,
+        dataStructureUtil.randomBytes32(),
+        dataStructureUtil.randomBytes32(),
+        dataStructureUtil.randomBytes(256),
+        specVersion.beaconStateAccessors().getRandaoMix(state, newEpoch),
```

Comment:
do we want a randomBytes256 since its in the data types now?

Reply:
Yeah why not? :)